frontend/test/__support__/cypress.js

```diff
@@ -290,4 +290,12 @@ function addQADatabase(engine, db_display_name, port) {
   }).then(({ status }) => {
     expect(status).to.equal(200);
   });
+
+  // Make sure we have all the metadata because we'll need to use it in tests
+  cy.request("POST", "/api/database/2/sync_schema").then(({ status }) => {
+    expect(status).to.equal(200);
```

Comment:
@paulrosenzweig This approach should work for `frontend/test/snapshot-creators/default.cy.snap.js:211` (withsqlite snapshot) instead of waiting arbitrarily, right?

Do I include that in this PR, and do you want me to even change that line during snapshot creation?